frame/tips/src/lib.rs

```diff
@@ -168,6 +170,8 @@ decl_event!(
 		TipClosing(Hash),
 		/// A tip suggestion has been closed. \[tip_hash, who, payout\]
 		TipClosed(Hash, AccountId, Balance),
+		/// A tip suggestion has been slashed. \[tip_hash, finder, deposit\]
```

Comment:
Probably best to keep this at the bottom of the list.

You have to think about how these structures and enums are encoded. At some point, there will be an event emitted with index 5 for example. If you place a new item at 4, then all other items shift forward by 1, and now 5 means something different. This isn't always important, and if you are using metadata correctly it shouldnt matter, but here, i dont see an advantage to introduce a breaking encoding change.

Reply:
Thanks to make me aware about encoding login behind this enums, moved the new one to end of the list. Done